.github/workflows/format.yml

```diff
@@ -0,0 +1,56 @@
+name: Format check
+
+on:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - '**.py'
+
+jobs:
+  generate:
```

Comment:
I think this name is part of the "user interface" so can it be a bit longer and more specific? `find-changed-files` or better...

Reply:
Updated